### ISSUE TITLE
Create 'Health check' component to be displayed in the ‘Docs’ tab in storybook

### ISSUE THREAD

**Issue by greenc05** [BBC-archive/psammead#4571]
**Is your feature request related to a problem? Please describe.**
WebCore use a component to display the health of a component in the ‘Docs’ tab in storybook. We already do the 'health checks' though don't display them as part of a component. Lets create our own version of this component. 

An example can be seen here of the WebCore component:
https://static.files.bbci.co.uk/core/storybook/index.html?path=/story/docs-components-health-factors--basic

Matadata for the WebCore component can be seen here: 
https://github.com/bbc/web/blob/main/packages/gel/components/promo/metadata.json  

The WebCore component is then used to display component health for WebCore components in the 'Docs' tab, an example can be seen here:
https://static.files.bbci.co.uk/core/storybook/index.html?path=/docs/gel-components-section-header--basic 

**Describe the solution you'd like**
- Create 'Health check' component to be displayed in the ‘Docs’ tab in storybook (Our own version of the WebCore 'Health Factors' component)
- The component should include 'Basic', 'Progress to done', 'With invalid data' and 'With no data'
- Spacing, font sizes, colours etc - Follow WebCore component
- Icon colours: Green: #0A7B0A, Replace yellow with: #3f3f42, Red: #D60A3D, Black: #3f3f42
- [Screen reader UX](https://paper.dropbox.com/doc/Health-check-Health-check-summary--BThYW6C_ycKhyhRLbz6HcGnjAg-y8T4oXU2Qe0ZjeQiwwnSE) 
- [Accessibility Acceptance Criteria](https://paper.dropbox.com/doc/Health-check-Health-check-summary-AAC--BTgPGZnrl~p5Fe0NhAfxYfTsAg-zxLRZFPMHGB5SQOtDB6NQ)  
- Text as per below screen shots
- This will require a 2 person developer accessibility swarm

Basic
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632935163698_Screenshot+2021-09-29+at+17.41.58.png)

Progress to done
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632935178690_Screenshot+2021-09-29+at+17.41.54.png)

With invalid data
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632935193322_Screenshot+2021-09-29+at+17.41.51.png)

With no data
![](https://paper-attachments.dropbox.com/s_B90994C53B45AE6E03D1EC3B3C5E5BDBA0E6997838FF525AA344E30F6DB81A11_1632935200073_Screenshot+2021-09-29+at+17.41.48.png)

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by blueputty01** [comment]
@greenc05 Is this issue still open given  PR #10343?

**Comment by greenc05** [comment]
Thanks @blueputty01 closing...
